### PR TITLE
Vault Page Updates

### PR DESCRIPTION
**Pull request by filippoweb3**
- [x] Support for Android
- [x] Additional features such as private key export etc.

### CODE REVIEW

**Comment by torabidm** [comment]
a few comments:
- there seems to be an android port of vault, haven't tested it though
- perhaps makes sense to link to official app stores rather than apk
- there is a comment on private key export, but the context is unclear how/why this could happen
- adding an account to the js app could be extended, e.g. saying only pubkey is stored by the extension, etc 
- perhaps that mnemonic phrase is created by the vault could be made explicit, or is there a way mnemonic phrases could be imported to the vault if a user wants to generate them themselves?

**Comment by filippoweb3** [comment]
@torabidm see the new additions, let me know what you think!

**Comment by torabidm** [comment]
thanks Filippo.. the changes look good to me

**Comment by filippoweb3** [comment]
@DrW3RK let me know if we can merge this PR

File: docs/general/polkadot-vault.md [hunk 1]
```diff
@@ -37,14 +41,20 @@ should consider one or the other depending on your user type.
...
+|         Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/). The secure element will detroy itself if opened.         |                                                                                                                                                                                                    Although there is no wired connection, side-channel attacks are possible. Without a secure element, the phone can be opened and the keys will be accessible in its storage unit.                                                                                                                                                                                                    |
```
Comment (profradha) [inline docs/general/polkadot-vault.md]:
```suggestion
|         Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/). The secure element will destroy itself if opened.         |                                                                                                                                                                                                    Although there is no wired connection, side-channel attacks are possible. Without a secure element, the phone can be opened, and the keys will be accessible in its storage unit.                                                                                                                                                                                                    |
```

File: docs/general/polkadot-vault.md [hunk 2]
```diff
@@ -37,14 +41,20 @@ should consider one or the other depending on your user type.
...
 | Ledger app updates on Ledger Live sometimes lag behind chain updates resulting in users only being able to transact if they [install developer versions](./ledger.md#install-the-developer-release) (only for advanced users). |                                                                                                                                                                                  Metadata updates are always available to install once released, either through the third-party provider or manually. In this case, the installation process requires familiarity with working on the command prompt.                                                                                                                                                                                  |
 
 ## Create and Import Accounts
 
+You can create a new account directly within the Vault app (Add Key Set > Add new Key Set). This
+will generate a new mnemonic seed phrase on the app. Alternatively, you can import a new account in
+the app using a seed phrase generated on a cold device elsewhere. For example, you can have an
```
Comment (profradha) [inline docs/general/polkadot-vault.md]: Can be a bit misleading. For instance, can we import the account created on Ledger?

```suggestion
the app using a seed phrase generated elsewhere with a compatible account generation scheme. For example, you can have an
```

File: docs/general/polkadot-vault.md [hunk 3]
```diff
@@ -56,6 +66,13 @@ to learn more about creating Vault accounts, and
 [this article](https://support.polkadot.network/support/solutions/articles/65000184118-polkadot-vault-how-to-add-your-account-on-polkadot-js-ui)
 on how to import them to the Polkadot extension or Polkadot-JS UI.
 
+:::info
+
+Importing an account into an extension will not import its private key. Only the public key will be
+imported and you must sign using your air-capped phone (as the private key is there).
```
Comment (profradha) [inline docs/general/polkadot-vault.md]:
```suggestion
imported and you must sign using your air-gapped phone which holds the private key.
```

**Review by DrW3RK — APPROVED** [review]
minor fixes

**Pull request merged**